### PR TITLE
Add documentation for Danfoss Air HRV.

### PR DESCRIPTION
**Pull request by JonasPed1**
**Description:**
Add documentation for Danfoss Air HRV component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#20138

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/danfoss_air.markdown [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: "Danfoss Air"
+description: "How to integrate Danfoss Air HRV in Home Assistant."
+date: 2019-01-14 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
```
Comment (rohankapoorcom) [inline source/_components/danfoss_air.markdown:9]: Please add a logo. You can add it to `source/images/supported_brands`

File: source/_components/danfoss_air.markdown [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ha_category: Hub
```
Comment (rohankapoorcom) [inline source/_components/danfoss_air.markdown]: This category doesn't seem right. I think this would be better to be a in `climate` or `fan`.

**Review by rohankapoorcom — CHANGES_REQUESTED** [review]
Please also add pages for the platforms included with this component.

**Comment by rohankapoorcom** [comment]
Looking like this is almost set - just needs pages for the platforms that are created as well.

**Comment by JonasPed1** [comment]
I was actually a bit unsure if documentation was needed for the individual platforms as no configuration for these is possible. I have updated PR with documentation for sensor and binary sensor platform.

**Comment by rohankapoorcom** [comment]
I think the new platforms documentation should indicate what things it provides sensors for.

**Comment by JonasPed1** [comment]
I think making a list with the sensors will make sense. Should this go in to the platform pages or the component page? Looks like it is a bit of a mix in the existing documentation?

**Comment by rohankapoorcom** [comment]
Platform pages makes more sense since there each platform exposes different sensors.

**Comment by JonasPed1** [comment]
Updated the platform pages with a short description of each sensor.

**Review by rohankapoorcom — APPROVED** [review]
This looks good to me. Thanks for working through it.

**Comment by rohankapoorcom** [comment]
@cgarwood I think this needs a version change to 0.87

**Comment by cgarwood** [comment]
Good catch. New PR to update version: #8249

**Pull request merged**